templates/search/search.html

```diff
@@ -4,6 +4,18 @@
 {% load paginator %}
 {% load search %}
 {% load util %}
+{% load static %}
+
+{% if clustering_on %}
+{% block extra_headers %}
+    <link rel="stylesheet" type="text/css" href="{{media_url}}css/graph-clustering.css"/>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
does this load a second jquery in addition to the one that is normally loaded on all pages? We should probably upgrade that one instead...